lib/datasource/types.ts

```diff
@@ -1,13 +1,16 @@
 import type { ModuleApi } from '../types';
 
-export interface Config {
+export interface GetDigestInputConfig {
   datasource?: string;
-  depName?: string;
   lookupName?: string;
+  depName: string;
   registryUrls?: string[];
+  currentValue?: string;
+  currentDigest?: string;
 }
 
-export interface DigestConfig extends Config {
+export interface DigestConfig {
```

Comment:
I've lost track of the difference between this and GetDigestConfig

Reply:
`GetDigestInputConfig` is raw input, `GetDigest` is sanitized version with resolved values, ready to be consumed from particular datasource. Our problem was that we tried to use same type for sanitized and non-sanitized config, which led to more redundant null-checks inside datasource implementations.